src/sftp/sftp.go

```diff
@@ -140,12 +142,19 @@ func (receiver *SftpHandler) CopyFiles() {
 		return
 	}
 
+	var wg sync.WaitGroup
```

Comment:
Should we add a comment explaining why we need a WaitGroup?

Reply:
good call, added a comment